src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -292,8 +288,14 @@ public boolean onTouch(View v, MotionEvent event) {
                     }
                 });
                 uploadToTextView.setText(String.format(getResources().getString(R.string.upload_to),getResources().getString(R.string.app_name)));
-                uploadBottomSheetBehavior = BottomSheetBehavior.from((View) uploadBottomSheet.getParent());
-                uploadBottomSheetBehavior.setPeekHeight(-1);
+                final BottomSheetBehavior uploadBottomSheetBehavior = BottomSheetBehavior.from((View) uploadBottomSheet.getParent());
+                uploadBottomSheetBehavior.setState(BottomSheetBehavior.STATE_EXPANDED);
```

Comment:
I think you don't need this line

Reply:
@davigonz Yes I forgot to remove it. Sorry I will remove it now.